[DOC] Update JMX connector documentation (#22248)
prestodb/presto: presto-docs/src/main/sphinx/connector/jmx.rst

Querying JMX
------------

The JMX connector provides three schemas: ``current``, ``history``, and ``information_schema``.

Current Schema
^^^^^^^^^^^^^^

The ``current`` schema contains every MBean from every node in the Presto

> cluster. To see all of the available MBeans, runn ``SHOW TABLES``::

Reviewer: ```suggestion
cluster. To see all of the available MBeans, run ``SHOW TABLES``::
```
I should have caught this last review.

Querying JMX
------------

The JMX connector provides three schemas: ``current``, ``history``, and ``information_schema``.

Current Schema
^^^^^^^^^^^^^^

The ``current`` schema contains every MBean from every node in the Presto

> cluster. You can see all of the available MBeans by running ``SHOW TABLES``::

Reviewer: ```suggestion
cluster. To see all of the available MBeans, runn ``SHOW TABLES``::
```

     2016-01-28 10:19:00.000 |  21422
     2016-01-28 10:19:10.000 |  31412
    (3 rows)

Information Schema
^^^^^^^^^^^^^^^^^^

The ``information_schema`` contains metadata about all the other schemas.

> You can see all the available tables in this schema by running the following command:

Reviewer: ```suggestion
To list all the available tables in this schema, run the following command:
```
